crawler/src/main/scala/StacImport.scala

```diff
@@ -75,7 +75,7 @@ class StacImport(val catalogRoot: String) {
   def readFromLocalPath(path: String): fs2.Stream[ConnectionIO, Json] =
     fs2.io.file
       .readAll[ConnectionIO](
-        Path.of(path),
+        Paths.get(path),
```

Comment:
I still don't understand why this didn't compile for me, but this does so hopefully it works for everyone.